Document/0x06d-Testing-Data-Storage.md

```diff
@@ -1,20 +1,21 @@
 ## Testing Data Storage
 
-### Testing Local Data Storage -- TODO [Merge with OWASP-DATAST-001-2] --
```

Comment:
I closed the issue assigned to it for you https://github.com/OWASP/owasp-mstg/issues/330